src/routes/+layout.svelte

```diff
@@ -147,4 +147,8 @@
         position: relative;
         pointer-events: all;
     }
+
+    :global(.web-input-text) {
+        --p-input-placeholder-text-color: var(--web-color-tertiary);
+    }
```

Comment:
this should rather be done in the scss files if its global anyway 👍🏻

Reply:
Done 👍.